### DISCUSSION TITLE
Open selected file in new window?

### DISCUSSION THREAD

**Discussion by ostrain** [prabirshrestha/vim-lsp#1110] [Q&A]
I'm sure this must be possible but I haven't been able to figure it out from the docs. When I run a command such as `:LspReferences` that displays a list of results, is it possible to configure vim-lsp to open the chosen result in a new window (i.e. a new vertical split)? It seems to always open the result in my rightmost window (maybe the most recently focused window?), which isn't usually what I want.

**Comment by tranvnhan** [comment]
I second this.
